Wappscript.js

```diff
@@ -0,0 +1,71 @@
+const windSpeedMap = new Map();
+const temperatureMap = new Map();
+
+
+fetch("http://localhost:8000/windspeed.json")
+            .then(response => response.json())
+            .then(data => {
+                fetch("http://localhost:8000/air-temperature.json")
+                    .then(response => response.json())
+                    .then(temperatureData => {
+                        
+                        const debugElement = document.querySelector("#debug");
+
+            
+
+                        for (let i = 0; i < data.items[0].readings.length; i++) {
+                            const windSpeedReading = data.items[0].readings[i];
+                            const temperatureReading = temperatureData.items[0].readings[i];
+
+                            const windSpeedStation = data.metadata.stations.find(station => station.id === windSpeedReading.station_id);
+                            const temperatureStation = temperatureData.metadata.stations.find(station => station.id === temperatureReading.station_id);
+
+                            const windSpeedStationName = windSpeedStation ? windSpeedStation.name : "Unknown Station";
+                            const temperatureStationName = temperatureStation ? temperatureStation.name : "Unknown Station";
+
+                            const windSpeedValue = windSpeedReading.value;
+                            const temperatureValue = temperatureReading.value;
+
+                            windSpeedMap.set(windSpeedStationName, windSpeedValue);
+                            temperatureMap.set(temperatureStationName, temperatureValue);
+
+
+                                
+                        
+                        }
+                        
+                            Sortbytemperature(temperatureMap);
+                            SortbywindSpeed(windSpeedMap);
+                       
+                        
+                        
+                    })
+                    .catch(error => {
+                        console.error("Temperature Fetch error:", error);
+                    });
+            })
+            .catch(error => {
+                console.error("WindSpeed Fetch error:", error);
+            });
+
+        function SortbywindSpeed() {
+            const sortedWindSpeedMap = new Map([...windSpeedMap.entries()].sort((a, b) => b[1] - a[1]));
```

Comment:
As far as I see, you are sorting the map here by values, is it in descending order or in ascending order? It will be good to add comments on top of the method to explain the sorting logic.